python/instrumentation/openinference-instrumentation-pydanticai/examples/pydantic.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+import phoenix as px
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+from pydantic import BaseModel
+from pydantic_ai import Agent
+from pydantic_ai.models.openai import OpenAIModel
+from pydantic_ai.providers.openai import OpenAIProvider
+
+from openinference.instrumentation.pydanticai import (
+    OpenInferenceSpanExporter,
+)
+
+# Get the secret key from environment variables
+os.environ["OPENAI_API_KEY"] = "YOUR_OPENAI_API_KEY"
+
+# Launch Phoenix app
+session = px.launch_app()
```

Comment:
I would probably avoid this pattern - it encourages bad practices. Just assume phoenix is running.